components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/impl/UserstoresApiServiceImpl.java

```diff
@@ -87,6 +88,14 @@ public Response getUserStoreManagerProperties(String typeId) {
         return Response.ok().entity(serverUserStoreService.getUserStoreManagerProperties(typeId)).build();
     }
 
+    @Override
+    public Response patchAttributeMappings(String userstoreDomainId,
```

Comment:
can we say `updateAttributeMappings` instead of `patchAttributeMappings`?

Reply:
Addressed with commit c731518fd0ea44d2d06c025d044e61114470